htmresearch/algorithms/apical_dependent_temporal_memory.py

```diff
@@ -44,7 +44,8 @@ def __init__(self,
                apicalInputSize=0,
                cellsPerColumn=32,
                activationThreshold=13,
-               initialPermanence=0.21,
+               reducedBasalThreshold=10,
+               initialPermanence=0.61,
```

Comment:
Let's use the same default `initialPermanence` as `ApicalTiebreakTemporalMemory` and `ApicalTMRegion`. The experiment code can override it to 0.61.